docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1,11 +1,11 @@
 ---
 slug: /en/operations/server-configuration-parameters/settings
 sidebar_position: 57
```

Comment:
This is the current order of doc pages:

<img width="333" alt="image" src="https://github.com/ClickHouse/ClickHouse/assets/100446237/22962601-7c32-444a-a14e-8233d34cd276">

In my view, a more logical order would be:
1. Settings Overview
2. Configuration files
3. User Settings
4. Settings Profiles
5. Constraints on Settings
6. Global Server Settings
7. Merge Tree Settings
8. Core Settings
9. Format Settings
10. Restrictions on Query Complexity

Pages "Permissions for Queries" and "Memory Overcommit" look out-of-place and is ideally integrated into an existing page.